CMakeLists.txt

```diff
@@ -156,13 +160,21 @@ if(ENABLE_CLANG_AS_LIB)
 
   set(VC_LIB_SRC
       ${VC_LIB_SRC}
-      ${SRC_PREFIX}/CompilerImpl/JITCompiler.cpp
       ${SRC_PREFIX}/CompilerImpl/ClangLibCompiler.cpp
       ${SRC_PREFIX}/CompilerImpl/ClangLLVM/FileLogDiagnosticConsumer.cpp
       ${SRC_PREFIX}/CompilerImpl/ClangLLVM/LLVMInstanceManager.cpp)
 
-  set(VC_LIB_HDR2 ${VC_LIB_HDR2} ${VC_LIB_HDR_PREFIX2}/JITCompiler.hpp
-                  ${VC_LIB_HDR_PREFIX2}/ClangLibCompiler.hpp)
+  if(ENABLE_JIT)
+    list(APPEND VC_LIB_SRC
+            ${SRC_PREFIX}/CompilerImpl/JITCompiler.cpp)
+  endif()
+
+  set(VC_LIB_HDR2 ${VC_LIB_HDR2} ${VC_LIB_HDR_PREFIX2}/ClangLibCompiler.hpp)
+
+  if(ENABLE_JIT)
+    list(APPEND VC_LIB_HDR2
+            ${VC_LIB_HDR_PREFIX2}/JITCompiler.hpp)
+  endif()
```

Comment:
These two `if` can be merged

Reply:
fixed in [c781515](https://github.com/skeru/libVersioningCompiler/pull/5/commits/c781515fa80507deedc8db01594ccf571865b2bd)